website/app/helpscout/page.tsx

```diff
@@ -1,21 +1,16 @@
+'use client';
 import React from 'react'
 import { LiveChatLoaderProvider, HelpScout } from 'react-live-chat-loader'
-import type { NextPage } from 'next'
+import ExampleLinks from '../../components/exampleLinks'
 
-import Layout from '../layouts/main'
-import ExampleLinks from '../components/exampleLinks'
-
-const Page: NextPage = () => (
-  <LiveChatLoaderProvider
-    provider="helpScout"
-    providerKey="b9ae6939-6c02-41de-b270-7f492379dc50"
-  >
-    <Layout title="React Live Chat Loader: Help Scout">
+export default function HelpScoutPage() {
+  return (
+    <LiveChatLoaderProvider provider="helpScout" providerKey="b9ae6939-6c02-41de-b270-7f492379dc50">
       <div className="wrapper">
         <div className="inner">
           <h1>React Live Chat Loader: Help Scout</h1>
           <p>
-            This is an example implementation of the Help Scout beacon using{' '}
+            This is an example implementation of the Help Scout widget using{' '}
```

Comment:
```suggestion
            This is an example implementation of the Help Scout beacon using{' '}
```

"beacon" is how Helpscout refer to their widget, so I'd opt to leave the language as-is